airflow/providers/http/operators/http.py

```diff
@@ -171,3 +184,103 @@ def execute_complete(self, context: Context, event: dict):
             return self.process_response(context=context, response=response)
         else:
             raise AirflowException(f"Unexpected error in the operation: {event['message']}")
+
+
+class ExtendedHttpOperator(SimpleHttpOperator):
```

Comment:
@Joffreybvn I'm not sure what, but the name could be improved. WDYT?

Reply:
Yep, commit was on its way for a renaming. Here it is !